Mathlib/Algebra/Order/BigOperators/GroupWithZero/List.lean

```diff
@@ -58,6 +58,23 @@ theorem prod_map_le_prod_map₀ {ι : Type*} {s : List ι} (f : ι → R) (g : 
       simp [ha]
     · apply (h0 _ _).trans (h _ _) <;> simp
 
+theorem prod_map_le_pow_length {F L : Type*} [MulPosMono R] [FunLike F L R] {f : F} {r : R}
```

Comment:
```suggestion
theorem prod_map_le_pow_length₀ {F L : Type*} [MulPosMono R] [FunLike F L R] {f : F} {r : R}
```
since the Group version of this should exist too, I think